### DISCUSSION TITLE
Switch to high-res stream when going full screen from live view

### DISCUSSION THREAD

**Discussion by peterjonesk** [blakeblackshear/frigate#12269] [Ideas]
Many tablets have trouble displaying the live view with a number of cameras (as pointed out by the experts in [this discussion](https://github.com/blakeblackshear/frigate/discussions/12225#discussion-6882412)). The video decoders just can not keep up. I've seen frequent or intermittent browser crashes on multiple tablets.

One current solution is to have the live screen show a low res substream (by setting "live -> stream_name"). But when clicking on the stream to go full screen, it still shows that same low res stream. Would be great to have it then switch to a high res stream.

Perhaps that setting could be reinterpreted to only apply to the live view showing multiple cameras. Or there could be another setting like "live_fullscreen -> stream_name".

**Comment by Paclanc** [comment]
I was just today thinking of this. Low res or substream on the live view to save bandwidth and make it more agile on mobile devices or even on desktop when viewing more than 10 cameras, but switch to the high res or main stream when selecting a camera to go fullscreen.

Where can we suggest this idea to the team?

**Reply by NickM-27** [reply]
We've thought about this quite a bit, question is mainly how best to implement it config wise

**Reply by mblaszczu** [reply]
I was wondering if there has been any progress on this feature? It would be a great improvement, especially in grid view, where the 4K interpolation to ~300x200 looks quite poor. Are there any plans to include this in the near future?

**Reply by hawkeye217** [reply]
Stream selection for camera groups and single camera live view will be a feature of Frigate 0.16.
